torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -32,6 +32,21 @@ glow::Type ptTypeToGlowType(const at::CompleteTensorType &ptType) {
   return glow::Type(glow::ElemKind::FloatTy, dims);
 }
 
+/// \returns a Glow tensor with the same type and underlying data as the given
+/// Pytorch tensor \p ptT.
+glow::Tensor ptTensorToGlowTensor(const at::Tensor &ptT) {
+  // TODO: get correct ElemKind.
+  std::vector<size_t> dims;
+  for (const auto d : ptT.sizes()) {
```

Comment:
```suggestion
  for (const auto &d : ptT.sizes()) {
```

Reply:
why? ref to int always seems weird to me

Reply:
Just personal preference. It's not a huge deal.